src/test/java/com/example/demo/services/lifebalancechecker/strategy/impl/LifeBalancerStrategySatisfyingOthersTest.java

```diff
@@ -0,0 +1,45 @@
+package com.example.demo.services.lifebalancechecker.strategy.impl;
+
+import com.example.demo.model.LifeBalance;
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LifeBalancerStrategySatisfyingOthersTest {
+
+    @Test
+    void balanceLife_whenCannotSatisfyOthers_returnsFalse() {
+        int meaningOfLife = 42;
+        double normalizedThresholdToSatisfyOthers = 0.01;
+        int tasksCountToCompleteBalance = 1000;
+        double desiredSelfCareRatio = 0.7;
+        LifeBalancerStrategySatisfyingOthers lifeBalancer = new LifeBalancerStrategySatisfyingOthers(
+                normalizedThresholdToSatisfyOthers,
+                tasksCountToCompleteBalance,
+                new Random(meaningOfLife),
```

Comment:
a ten Random to nie jest jakas zew. zależność, zwróci zawsze to samo?

Reply:
w takiej postaci zwróci to samo. W testach przekazuję randoma ze stałym seedem